derivations/low_mach_navier_stokes_jacobians/low_mach_navier_stokes.tex

```diff
@@ -0,0 +1,128 @@
+\subsection{Background Information}
+The derivation detailed herein is for computing analytical Jacobians for the Low Mach Navier-Stokes equations. The final results are then implemented in GRINS src/physics/src/low\_mach\_navier\_stokes.C, with the goal being that there
+will be a significant speedup in computation over using the standard finite difference approximation for computing the Jaconians. The derivation is broken up into 3 subsections: mass (P residual), momentum (U residual), and energy (T residual).
+Each of these is computed in a separate function wihtin the LMNS.C file.  The 3 residuals were already implemented, so they will be the starting point for this derivation. Note that a sign convention has been specified for 
+purposes of optimization.\\
+Also note that, in this case, the following parameters are considered functions of temperature, T:
+\begin{itemize}
+    \item density, $\rho$
+    \item specific heat, $c_p$
+    \item viscosity, $\mu$
+    \item thermal conductivity, $k$
+\end{itemize}
+For the formulations, \textbf{u}=\{$P,U,T$\} and the test functions \textbf{$\Phi$}=\{$\xi,\phi,\psi$\}.\\
+Lastly, the variable names used in the code follow this template: $Kxy$, where $x$ is the residual and $y$ is the derivation variable.  For example, $KPT$ is the derivative of the P residual with respect to temperature,
+while $KuP$ is the derivative of the $u$-velocity component of the U residual with respect to pressure. 
+
+\newpage
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection{Mass Equation}
+The given P residual:
+\begin{equation}
+    F_p = (-(U \cdot \nabla T) T^{-1} + \nabla \cdot U)\xi
+\end{equation}
+
+Now converting to tensor notation
+
+\begin{equation}
+    F_p = -u_a T_{a,b} T^{-1} \xi + u_{a,b}\xi
+\end{equation}
+
+Galerkin approximation:
+\begin{align*}
+    u_a &= \sum u_i \phi_i^a\\
+    T &= \sum T_i \psi_i
+\end{align*}
+
+\begin{equation}
+    F_p = -u_i \phi_i^a T_j \psi_j^{a,b} (T_k \psi_k)^{-1} \xi + u_i \phi_i^{a,b} \xi
```

Comment:
If you're using the summation convention, be sure repeated indices don't appear more than once. Also, I like `i` to be used for the test function.